### DISCUSSION TITLE
cs/tcs/toc/topic3/

### DISCUSSION THREAD

**Discussion by giscus** [TonyCrane/note#40] [General]
# cs/tcs/toc/topic3/

鹤翔万里（TonyCrane）的笔记本

https://note.tonycrane.cc/cs/tcs/toc/topic3/

**Comment by EclipseYue** [comment]
图灵机形式化表达右移的情况里$\u_1 = u_2 a_2$是不是写反了？应该是$u_1 = a_2 u_2$

**Comment by TillyEndless** [comment]
变种图灵机δ:(K−H)×Σ^k→K×((Σ−{⊳})∪{←,→})似乎最后一个括号也是k的幂次，即
δ:(K−H)×Σ^k→K×((Σ−{⊳})∪{←,→})^k
